### PR TITLE
implemented feature add assignment

### PR DESCRIPTION
**Pull request by ian-lowe-csumb**
implemented feature 'add assignment'

the new 'add' button opens a modal which lets the user input an assignment name, due date, and course id

for now, the date input field is just a string since the datepicker package for material UI wasn't included in the code provided to us

### CODE REVIEW

File: src/components/Assignment.js [hunk 1]
```diff
@@ -13,14 +18,56 @@ import {SERVER_URL} from '../constants.js'
...
+    this.setState({ open:false });
+  };
+
+  handleChange = (event) => {
+    this.setState({[event.target.name]: event.target.value});
+  }
+
+  handleSubmit = () => {
+    this.addAssignment(this.state.assignmentName, this.state.dueDate, this.state.courseId);
+    this.setState({ open:false });
```
Comment (sabrinafair) [inline src/components/Assignment.js:44]: or could send to this.handleClose() to prevent repeat.

File: src/components/Assignment.js [hunk 2]
```diff
@@ -13,14 +18,56 @@ import {SERVER_URL} from '../constants.js'
...
+      {  
+        method: 'POST', 
+        headers: { 
+          'X-XSRF-TOKEN': token,
+          'Accept': 'application/json',
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({assignmentName: name, dueDate: dueDate})
+      } )
+    .then((response) => {
```
Comment (sabrinafair) [inline src/components/Assignment.js:60]: It might not be necessary, but you could put a toast.success here for when the assignment is successfully added

Comment (ian-lowe-csumb) [reply src/components/Assignment.js:60]: Added

**Comment by sabrinafair** [comment]
LGTM!

**Pull request merged**